runtime/src/lib.rs

```diff
@@ -126,11 +125,12 @@ pub mod opaque {
     pub type BlockId = generic::BlockId<Block>;
 
     impl_opaque_keys! {
-        pub struct SessionKeys {}
+        pub struct SessionKeys {
+		pub aura: Aura,
+	    }
     }
 }
 
-pub use constants::time::*;
 pub use types::*;
 
 pub const VERSION: RuntimeVersion = RuntimeVersion {
```

Comment:
should this be renamed different to the standalone chain (i.e. `datahighway-parachain`)?

Reply:
should `impl_version` be `0` instead of `1`?

* Read both of these as comments are scattered:
        * Knowledgebase https://substrate.dev/docs/en/knowledgebase/runtime/upgrades#runtime-versioning * Substrate API Docs code comments https://substrate.dev/rustdocs/v3.0.0/substrate_test_runtime_client/sc_executor/struct.RuntimeVersion.html#structfield.spec_version
        * Other comments in Substrate codebase say:
            * bug fixes
                * should result in an increment of spec_version and possibly authoring_version,
                * absolutely not impl_version since they change the semantics of the runtime.
            * runtime behavior changes
                * increment spec_version and set impl_version to 0
            * only runtime implementation changes and behavior does not change
                * then leave spec_version as is and increment impl_version.